tests/test-authentication-client/src/main/resources/keycloak.json

```diff
@@ -1,7 +1,7 @@
 {
   "realm": "rhamt",
   "auth-server-url": "http://localhost:8180/auth",
-  "resource": "rhamt-web/api",
+  "resource": "rhamt-web",
```

Comment:
ah, I missed this one. Even though I fixed it almost everywhere else... Thanks :)